src/uct/ib/dc/dc_mlx5.h

```diff
@@ -366,15 +391,45 @@ void uct_dc_mlx5_iface_set_ep_failed(uct_dc_mlx5_iface_t *iface,
 
 void uct_dc_mlx5_iface_reset_dci(uct_dc_mlx5_iface_t *iface, uint8_t dci_index);
 
+ucs_status_t uct_dc_mlx5_iface_create_dci(uct_dc_mlx5_iface_t *iface,
+                                          uint8_t pool_index, uint8_t dci_index,
```

Comment:
1. can we pass dci_config instead of all these params?
2. is pool_index still needed?